database/buildSchema/30_review_status_history.sql

```diff
@@ -0,0 +1,37 @@
+-- review section join
+
+
+CREATE TYPE public.review_status as ENUM ('Draft', 'Submitted', 'Changes Required', 'Review Pending');
+
+CREATE TABLE public.review_status_history (
+	id serial primary key,
+	review_id integer references public.review(id),
+	"status" public.review_status,
+	"timestamp" timestamp with time zone,
```

Comment:
```suggestion
	timestamp timestamp default current_timestamp,
```
Same as above

Reply:
Added default, but kept timestamp with timezone, also added default to others:

In the commit you'll see that existing timestamps are with time zone.

beb49d9183bf6fd8f07b38e9a8ded29437cd4a1a

Reply:
Haha, I actually changed them to remove the timezone in another branch that hasn't been merged yet. 😕

We should probably agree on this, but personally, I don't see why you'd ever want timezone info anywhere near a timestamp on a server, that's just asking for problems down the line, isn't it?

Reply:
it's pretty standard I thought, to record timestamp on the server, so that it can be displayed properly on front end (server in Cuba, front end is served in Lao, server saves timestamps on creation <- would be server time, but we want to save timezone so that when we display it in Lao, it's in Lao time at the time of creation/submission)

Reply:
From my understanding, timezones on server are a world of pain. If the server (and all internal processing and time logic) is ALWAYS UTC, then you know all time calculations will be fine with no adjustments needed. It's only when you go to display a time to the user, you convert the UTC to the user's local time.  That's by far the simplest and safest option IMHO. 

And this opinionated post agrees: http://yellerapp.com/posts/2015-01-12-the-worst-server-setup-you-can-make.html 😆

(I dealt with quite a few issues in mSupply earlier in the year that were caused by exactly this kind of timezone problem -- it's not fun).